buildsystem/dependencies.gradle

```diff
@@ -21,6 +21,7 @@ ext{
     androidx_viewmodel = "2.2.0"
     glide = "3.7.0"
     recyclerview = "26.1.0"
+    material = "1.0.0-rc01"
```

Comment:
be careful do not use rc versions.. use 1.0.0 or 1.0.1 or the most recent

Reply:
Okey, done!